src/main/xerus/monstercat/tabs/TabSettings.kt

```diff
@@ -90,6 +94,13 @@ class TabSettings: VTab() {
 		})
 		addLabeled("Player Coverart priorities:", createComboBox(Settings.PLAYERARTPRIORITY))
 		
+		// Export chooser
+		val exportFileChooser = FileChooser(App.stage, Settings.PLAYEREXPORTFILE().toFile(), "", "export file").apply { selectedFile.listen { Settings.PLAYEREXPORTFILE.set(it.toPath()) } }
+		addRow(Label("Export currently played track :"), exportFileChooser.button().allowExpand(vertical = false), exportFileChooser.textField())
+		
+		val connectionSpeed = createComboBox(Settings.CONNECTIONSPEED)
```

Comment:
what is this doing in here? a merge fail maybe?

Reply:
Probably, I will revert.

Reply:
Removed with ca3baa1